netbox/dcim/signals.py

```diff
@@ -1,15 +1,15 @@
 import logging
 
-from django.db.models.signals import post_save, post_delete
+from django.db.models.signals import post_delete, post_save
 from django.dispatch import receiver
 
 from dcim.choices import CableEndChoices, LinkStatusChoices
-from virtualization.models import VMInterface
-from .models import (
-    Cable, CablePath, CableTermination, ConsolePort, ConsoleServerPort, Device, DeviceBay, FrontPort, Interface,
-    InventoryItem, ModuleBay, PathEndpoint, PowerOutlet, PowerPanel, PowerPort, Rack, RearPort, Location,
-    VirtualChassis,
-)
+from ipam.models import Prefix
+from virtualization.models import Cluster, VMInterface
+from wireless.models import WirelessLAN
+from .models import (Cable, CablePath, CableTermination, ConsolePort, ConsoleServerPort, Device, DeviceBay, FrontPort,
+                     Interface, InventoryItem, Location, ModuleBay, PathEndpoint, PowerOutlet, PowerPanel, PowerPort,
+                     Rack, RearPort, Site, VirtualChassis)
```

Comment:
Revert formatting change
```suggestion
from .models import (
    Cable, CablePath, CableTermination, ConsolePort, ConsoleServerPort, Device, DeviceBay, FrontPort, Interface,
    InventoryItem, Location, ModuleBay, PathEndpoint, PowerOutlet, PowerPanel, PowerPort, Rack, RearPort, Site,
    VirtualChassis,
)
```